.github/workflows/main.yml

```diff
@@ -51,7 +51,7 @@ jobs:
                 id: test-coverage
                 uses: johanvanhelden/gha-clover-test-coverage-check@v1
                 with:
-                    percentage: "95"
+                    percentage: "99"
```

Comment:
Why not 100? :100:

Reply:
Does it compare by `>` or by `>=`? :)

Reply:
Docs says "minimum percentage of coverage allowed" so I think it compares by `>` :thinking: